### PR TITLE
Use an ellipsis for long channel names on desktop.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review. I don't know if it's the best approach, but it works.

### CODE REVIEW

File: shared/common-adapters/text.meta.desktop.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const lineClamp = (lines: number) => ({
   textOverflow: 'ellipsis',
   WebkitBoxOrient: 'vertical',
   WebkitLineClamp: lines,
+  wordBreak: 'break-all',
```
[the remark this answers is not in the document]
Comment (chrisnojima) [reply shared/common-adapters/text.meta.desktop.js]: mmmmmmmm, maybe ok but might break something. i think we can add it and fix things up

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

File: shared/common-adapters/text.meta.desktop.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (buoyad) [reply shared/common-adapters/text.meta.desktop.js]: hm, I think this broke lineclamping for long teamnames on the desktop inbox

[the remark this answers is not in the document]
Comment (adamjspooner) [reply shared/common-adapters/text.meta.desktop.js]: For anyone following along: https://github.com/keybase/client/pull/14731

**Pull request merged**